pkg/kudoctl/util/repo/repo.go

```diff
@@ -57,6 +59,20 @@ func (r Repositories) CurrentRepo() *RepositoryConfiguration {
 	return r.GetRepo(r.Context)
 }
 
+// RepositoryConfig gets the repo configuration from settings
+func RepositoryConfig(fs afero.Fs, settings *env.Settings) (*RepositoryConfiguration, error) {
+	r, err := LoadRepositories(fs, settings.Home.RepositoryFile())
+	if err != nil {
+		// this allow for no client init... perhaps we should return the error requesting kudo init
+		r = NewRepoFile()
```

Comment:
this is fine, are there other errors that could come back from this? if so, let's have a specific error check function for it. in fact, I'd prefer it _anyway_ since I could totally see this blowing up in the future as LoadRepositories grows. especially thinking about my other code review comment in #751 here as an example - what about permissions errors in LoadRepositories?